### PR TITLE
Bring back changes from `release-v1.3.0` into `main`

### PR DESCRIPTION
**Pull request by curquiza**
Using a temp branch because of git conflict

### CODE REVIEW

**Comment by curquiza** [comment]
⚠️ DON'T MERGE IT, I just saw GitHub mess up with the README, I have to fix it before merging

**Review by curquiza — COMMENTED** [review]
✅ Ok for me for the README

**Review by ManyTheFish — APPROVED** [review]
bors merge

**Pull request merged**
